### ISSUE TITLE
[Localization] Toggling device from supported non-English language to unsupported does not default to English

### ISSUE THREAD

**Issue by rhsalas3** [willowtreeapps/eyespeak-ios#673]
**Describe the bug**
It is expected that if my device's language is set to an unsupported non-English language (Japanese for example) then the localization will default to English. However, if my device is initially set to any supported non-English language (Spanish or German, for example) and then set to an unsupported language what is the expected behavior? Currently the app displays text in the supported non-English language, instead of defaulting to English.

This is the behavior regardless of whether English is listed as one of the device's supported languages; w/in Settings > General > Language and Region > list of preferred languages

**Steps to Reproduce:**
1. Set the language of your test device to a supported non-English language; Spanish.
2. Install and launch Vocable; note that text is in Spanish.
3. Quit Vocable (kill the app; not just background it).
4. Open the device settings and toggle to an unsupported non-English language; Japanese
5. Open Vocable
6. Note that the text is displayed in Spanish

**Expected behavior**
?? This needs to be further defined by the team, but at the time of this writing I would expect the text to default to English.

**Actual behavior**
The text defaults to the previous language instead of defaulting to English.

**Device Information**

 - Device: iPhone 11 (simulator)
 - OS: iOS 15.5

https://user-images.githubusercontent.com/34660497/169065859-fe143241-2f50-4092-af6e-51bbf470e44c.mp4

**Comment by jmilillo** [comment]
I don't think this is an issue. On iOS, the language list in Settings is treated as priority.  In this case Spanish was the second language in the list, so it was chosen because it was supported.  If we set the order to Japanese, English, Spanish - English will be chosen.
